src/main/java/frc/robot/subsystems/drive/DriveCal.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot.subsystems.drive;
+
+import edu.wpi.first.math.controller.PIDController;
+import frc.robot.Constants;
+
+public class DriveCal {
+    public static final int SPARK_INIT_RETRY_ATTEMPTS = 5;
+
+    /** Input meters/second, output [-1,1] */
```

Comment:
Are the units for this the same as the units were in Rev-world?

Reply:
`TalonFX.setControl` (L190 in SwerveModule.java) takes an input [-1,1]; I *think* that's the place to look, in which case this is correct?